pmlc/ast/builder.cc

```diff
@@ -752,6 +768,106 @@ struct ProgramBuilder {
         .result();
   }
 
+  Value handleLoop(ExprNodeLoop *node) {
+    // take lowBound, upperBound, step from operands.
+    auto indexs = node->indexs;
+    if (indexs.size() != 3) {
+      throw std::runtime_error(
+          "please check the index of loop, lbound, ubound and step");
```

Comment:
Can we move this check earlier in ast.cc? Frank made comments in previous PR that suggest us to warn users earlier so they know it is their input not meet the API, not a bug in the compiler.

Reply:
Actually,  since we changed the `loop` API, I think this check is already useless, I will remove this one.